### PR TITLE
Add functions to calculate quantile based sensitivity measures.

### PR DESCRIPTION
**Pull request by Yuleii**

### CODE REVIEW

**Review by loikein — CHANGES_REQUESTED** [review]
Per [PEP](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), function or variable name should contain no upper case letter, please fix all instances.

Also, please read the checks output carefully, and try to let them pass. (Your local repo may have not properly installed pre-commit.)

**Comment by loikein** [comment]
My branch is merged, you can proceed now.

File: econsa/tests/test_quantile_measures.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+from scipy.stats import norm
+
+from econsa.quantile_measures import mcs_quantile
+
+
+@pytest.fixture
+def first_example_fixture():
+    """First example test case."""
+
+    def func1(args):
```
Comment (peisenha) [inline econsa/tests/test_quantile_measures.py:18]: Glad to see you getting started with tests, please use (or add if neeed) all test functions to  `temfpy`. @loikein  can help in the process.

Comment (Yuleii) [reply econsa/tests/test_quantile_measures.py:18]: Ok!

**Comment by peisenha** [comment]
Thank you! However, also please add documentation right away as well. Talk to you more tomorro, Philipp

---

Philipp Eisenhauer
Economist

Mail [peisenha@protonmail.com](mailto:eisenhauer@policy-lab.org)
Web https://peisenha.github.io
Repository https://github.com/peisenha

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Review by loikein — CHANGES_REQUESTED** [review]
Please make the plotting cells in tutorial notebook hidden, you can consult existing notebooks for reference.

**Review by loikein — APPROVED** [review]

File: econsa/tests/test_quantile_measures.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+def test_wrong_value_criterion(test_1_fixture):
+    """Make sure an error is raised if an argument has a wrong value."""
+    func = test_1_fixture["func"]
+    n_params = test_1_fixture["n_params"]
+    loc = test_1_fixture["loc"]
+    scale = test_1_fixture["scale"]
+    dist_type = test_1_fixture["dist_type"]
+
+    with pytest.raises(ValueError):
+        mc_quantile_measures(
```
Comment (peisenha) [inline econsa/tests/test_quantile_measures.py:110]: Please see if you can freeze the arguments that do not change between calls using `partial`.  Here are some details: https://www.learnpython.org/en/Partial_functions ... also applicable a little later in this module.

**Review by peisenha — APPROVED** [review]
Thanks! Please just see if you can address the issue below.  Then @loikein , please just go ahead and merge, delete branch directly. No need to check back in.

File: econsa/tests/test_quantile_measures.py (hunk 2 above)
Comment (Yuleii) [reply econsa/tests/test_quantile_measures.py:110]: Thanks! I will do that.

File: econsa/tests/test_quantile_measures.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    return out
+
+
+def test_1(test_1_fixture):
+    norm_q_2_true = test_1_fixture["norm_q_2_true"]
+    func = test_1_fixture["func"]
+    n_params = test_1_fixture["n_params"]
+    loc = test_1_fixture["loc"]
+    scale = test_1_fixture["scale"]
+    dist_type = test_1_fixture["dist_type"]
```
Comment (loikein) [inline econsa/tests/test_quantile_measures.py:79]: Please make the code more concise, I think you can use `*args` and `**kwargs` here. Or just use the fixture as a whole.

File: econsa/tests/test_quantile_measures.py [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+    ):
+        norm_q_2_solve = mc_quantile_measures(
+            estimator=estimator,
+            func=func,
+            n_params=n_params,
+            loc=loc,
+            scale=scale,
+            dist_type=dist_type,
+            n_draws=n_draws,
+        )
```
Comment (loikein) [inline econsa/tests/test_quantile_measures.py:94]: Did you try to use `partial` in more places? For example this looks like fit for me.

**Review by loikein — CHANGES_REQUESTED** [review]

File: econsa/tests/test_quantile_measures.py (hunk 3 above)
Comment (Yuleii) [reply econsa/tests/test_quantile_measures.py:79]: Ok!

**Review by loikein — APPROVED** [review]

**Pull request merged**